newIDE/app/src/guidelines/index.js

```diff
@@ -0,0 +1,205 @@
+// @flow
+import React, { PureComponent } from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@material-ui/core/Button';
+import Popper from '@material-ui/core/Popper';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import guidelines from './data.js';
+import Window from '../Utils/Window';
+import { getHelpLink } from '../Utils/HelpLink';
+import { Line } from '../UI/Grid';
+
+//TODO wrap l'image dans le Paper avec l'overflow hidden
+//Actuellement l'image dépasse du Paper alors qu'il y a une marge autour du Popper
+//Actualisé la position de la Popper.
+
+const styles = {
+  container: {
+    maxWidth: 400,
+    padding: 10,
+    marginTop: 10,
+    overflow: 'hidden',
+  },
+  description: {
+    paddingBottom: 10,
+  },
+};
+
+type State = {|
+  open: boolean,
+  index: number,
+  indexData: number,
+  indexMax: number,
+  anchor: *,
+  arrowRef: *,
+|};
+
+type Props = {|
+  open: boolean,
+  closeHandler: () => void,
+|};
+
+//TODO
+//PureComponent ou React.Component<Props, State> voir la doc pour comprendre la différence.
```

Comment:
Consider PureComponent as a way to optimize the rendering. If your component is only changing when the props are changing, use PureComponent. Otherwise, stick to Component.